learn-git

```diff
@@ -0,0 +1 @@
+Subproject commit c22435da3ce8b925ab0348ffd8c576378acfeaf3
```

Comment:
Uh oh... you should talk to a DIR or instructor.  You've never seen this before but I have...and it's scary.  It means one git repo is inside of another.